packages/platform/src/lib/platform-plugin.ts

```diff
@@ -31,6 +31,6 @@ export function platformPlugin(opts: Options = {}): Plugin[] {
     (platformOptions.ssr
       ? devServerPlugin({ entryServer: opts.entryServer })
       : false) as Plugin,
-    ...angular(opts?.vite),
+    ...angular({ jit: platformOptions.jit, ...opts?.vite }),
```

Comment:
If `opts?.vite` ends up being undefined this spread will error. I think you want `...(opts?.vite ?? {})`?

Reply:
Good catch, thanks